src/Provider/GoogleMaps/Model/SubLocalityLevel.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Geocoder package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Provider\GoogleMaps\Model;
+
+/**
+ * Class SubLocalityLevel is used only for GoogleMap provider and contains functions for working with concrete subLocalityLevel
+ */
+final class SubLocalityLevel
```

Comment:
My first idea was to use `AdminLevel` & `AdminLevelCollection`. I actually make it that way, but  `AdminLevel` class has hardcoded word `adminLevel`. I dont think it is ok to have subLocalityLevel called adminLevel in the result.. So I decided to make new classes.